app/src/main/res/values/strings.xml

```diff
@@ -1380,7 +1380,7 @@
     <!--The camera projection type for augmented reality (ex. perspective)-->
     <string name="projection_type">Augmented reality projection</string>
     <!--Camera projection using intrinsics matrix-->
-    <string name="projection_estimated_intrinsics">Estimated Intrinsics</string>
+    <string name="projection_estimated_intrinsics">Estimated intrinsics</string>
     <!--Camera projection using intrinsics matrix-->
     <string name="projection_manufacturer_intrinsics">Manufacturer Intrinsics</string>
```

Comment:
This should be sentence cased to match the estimated intrinsics

Reply:
you mean title case? as it is already sentence case